nemo_curator/utils/fuzzy_dedup_utils/merge_utils.py

```diff
@@ -36,6 +34,16 @@ def _split_part(part, nsplits):
 
 
 def text_bytes_aware_merge(text_df, right_df, broadcast=True, *, on):
```

Comment:
Is this function currently used anywhere?

Reply:
I don't see any usage of this function. cc: @VibhuJawa in case you recall any background and if this is needed or not

Reply:
Dont think this is needed anymore. 

We previously added it while trying to be more conservative IIRC

https://github.com/rapidsai/rapids-deduplication/pull/68